css/at-rules/supports.json

```
@@ -44,6 +44,76 @@
             "deprecated": false
           }
         },
+        "font-format": {
+          "__compat": {
+            "description": "<code>font-format()</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": false,
```

```suggestion
              "edge": {
                "version_added": false
              },
```

Same for the others.

---

ah-ha, thanks for the hint